src/components/Pagination/tests/Pagination.test.tsx

```diff
@@ -78,6 +78,33 @@ describe('<Pagination />', () => {
     });
   });
 
+  describe('accessibilityLabel', () => {
+    it('inserts prop as aria-label', () => {
+      const accessibilityLabel = 'test';
+      const pagination = mountWithAppProvider(
+        <Pagination accessibilityLabel={accessibilityLabel} />,
+      );
+      expect(pagination.find('nav').prop('aria-label')).toStrictEqual(
+        accessibilityLabel,
+      );
+    });
+
+    it('uses default value for aria-label', () => {
+      const pagination = mountWithAppProvider(<Pagination />);
+      expect(pagination.find('nav').prop('aria-label')).toStrictEqual(
+        'Pagination',
+      );
+    });
+  });
+
+  describe('label', () => {
+    it('renders as text', () => {
+      const label = 'test';
+      const pagination = mountWithAppProvider(<Pagination label={label} />);
+      expect(pagination.text()).toContain(label);
+    });
+  });
```

Comment:
Had you considered adding a test to verify that the label is `subdued` when there are no next or previous pages 🤔

Reply:
Done! 👍